streams/src/main/java/org/apache/kafka/streams/kstream/ValueTransformer.java

```diff
@@ -44,6 +44,8 @@
  * @param <VR> transformed value type
  * @see ValueTransformerSupplier
  * @see ValueTransformerWithKeySupplier
+ * @see KStream#transformValues(ValueTransformerSupplier, String...)
+ * @see KStream#transformValues(ValueTransformerWithKeySupplier, String...)
```

Comment:
This should be `KTable` right?

Reply:
Yes! Sorry, I have missed this after `git reset`.